doc/sphinx/conf.py

```diff
@@ -0,0 +1,400 @@
+#!/usr/bin/env python3
+##########################################################################
+##         #   The Coq Proof Assistant / The Coq Development Team       ##
+##  v      #   INRIA, CNRS and contributors - Copyright 1999-2018       ##
+## <O___,, #       (see CREDITS file for the list of authors)           ##
+##   \VV/  ###############################################################
+##    //   #    This file is distributed under the terms of the         ##
+##         #     GNU Lesser General Public License Version 2.1          ##
+##         #     (see LICENSE file for the text of the license)         ##
+##########################################################################
+# -*- coding: utf-8 -*-
```

Comment:
That coding line should be above the license header

Reply:
Thanks, will fix! Out of curiosity, what uses it?

Reply:
Python and Emacs